### PR TITLE
Stop monitoring when stdin shuts down

### PR DESCRIPTION
**Pull request by josevalim**
This allows us to use inotify-wait as a child process
and also allows us to shut it down on cmd by sending
Ctrl+Z.

### CODE REVIEW

**Comment by thekid** [comment]
Good suggestion!

File: src/Runner.cs [hunk 1]
```diff
@@ -110,6 +110,12 @@ protected void Output(TextWriter writer, string[] tokens, FileSystemWatcher sour
 			writer.WriteLine();
 		}
 
+		public void Stop(object data) {
+			while(Console.ReadLine() != null){ };
```
Comment (thekid) [inline src/Runner.cs]: Hrm - is there a more elegant way to do this?

Comment (josevalim) [inline src/Runner.cs]: Afaik, there isn't. :)

**Comment by thekid** [comment]
I've merged this after making some slight adjustments. While implementing this I came up with an interesting usecase:

```sh
$ sleep 10 | inotifywait.exe ../directory/to/monitor
# Checks whether either 10 seconds have passed or an event was triggered in the directory
```

**Comment by josevalim** [comment]
Thank you!

**Pull request closed without merging**